Firebase.podspec

```diff
@@ -95,12 +95,13 @@ Simplify your app development, grow your user base, and monetize more effectivel
 
   s.subspec 'AppDistribution' do |ss|
     ss.dependency 'Firebase/CoreOnly'
-    ss.ios.dependency 'FirebaseAppDistribution', '~> 9.6.0-beta'
+    ss.ios.dependency 'FirebaseAppDistribution', '~> 10.0.0-beta'
+    ss.ios.deployment_target = '11.0'
   end
 
   s.subspec 'AppCheck' do |ss|
     ss.dependency 'Firebase/CoreOnly'
-    ss.dependency 'FirebaseAppCheck', '~> 9.6.0'
+    ss.dependency 'FirebaseAppCheck', '~> 10.0.0'
     ss.ios.deployment_target = '10.0'
     ss.osx.deployment_target = '10.13'
     ss.tvos.deployment_target = '11.0'
```

Comment:
12.0 here and everywhere for tvos and in CHANGELOG

Reply:
tvos: 0527f1a
changelog: 94b267c